index.html

```diff
@@ -11,22 +11,48 @@
     <header>
     <nav class="main-container">
       <a href="#" class="main-navbar-logo">John Doe</a>
-      <a href="#" class="main-navbar-menu logo-right-whole">
-        <img src="./images/Union.png" alt="Menu"/>
-        <div class="righthand-of-logo righthand-of-logo-2">
-          <h3 class="porfolio-logo logo-portfolio-right-content">
-              Portfolio
-          </h3>
-          <h3 class="about-about logo-portfolio-right-content">
-              About
-          </h3>
-          <h3 class="contact-logo logo-portfolio-right-content">
-              Contact
-          </h3>
-          <img class="logo-portfolio-right-content" src="./images/Union2.svg" alt="Envelop Icon">
-       </div>
+      <a href="#" class="main-navbar-menu logo-right-whole main-navbar-menu-2">
+        <img class="menu" src="./images/Union.png" alt="Menu">
       </a>
-    </nav>
+
+
+      <aside class="sidebar ">
+        <!-- <div class="righthand-of-logo righthand-of-logo-2"> -->
```

Comment:
- [ ] Please, remove this piece of old commented-out code to improve the cleanliness of the project. 💪